userappwithbroker/build.gradle

```diff
@@ -52,5 +68,10 @@ android {
 dependencies {
     // Compile Dependencies
     implementation "com.android.support:design:$rootProject.ext.supportLibraryVersion"
-    implementation project(':adal')
+
+    localImplementation(project(':adal'))
+    distImplementation("com.microsoft.aad:adal:1.16.3-RC10") {
```

Comment:
Update this in `dev`

Reply:
Issue filed here: https://github.com/AzureAD/azure-activedirectory-library-for-android/issues/1433